packages/office-ui-fabric-react/src/components/Theme/docs/ThemesOverview.md

```diff
@@ -1,33 +1,38 @@
 The entire color palette of the controls are themeable. We provide a set of sensible defaults, but you can override all colors individually.
```

Comment:
"Themable" without the E in the middle.

Also, we should have a link on this page to the theme generator.

Reply:
Not ideal, since it's a little more optimized for editing source vs just viewing, but we could create an aka.ms link with a link to its absolute GitHub link, e.g. https://github.com/OfficeDev/office-ui-fabric-react/edit/master/packages/office-ui-fabric-react/src/components/Theme/docs/ThemesOverview.md. 

We do this for "Edit" links to control documentation:
![image](https://user-images.githubusercontent.com/307118/47029254-f803f380-d11f-11e8-8834-bcecca95a04b.png)

Reply:
I think we want to do a link *from* the MD to the color customization page (https://developer.microsoft.com/en-us/fabric#/styles/themegenerator). Are there aka.ms links for any of the developer pages?

Reply:
Also, I don't think "themable" is a word. 😆   http://www.yourdictionary.com/themeable

Reply:
@JasonGore we've been using "themable" for a long long time now:
https://docs.microsoft.com/en-us/sharepoint/dev/general-development/how-to-make-custom-css-files-themable-in-sharepoint

Bing gets more hits on "themEable" than "themable", but Google is the opposite.
And .NET uses "themEable" instead.
https://docs.microsoft.com/en-us/dotnet/api/system.web.ui.themeableattribute?view=netframework-4.7.2

In summary, I have no idea what to do =)
We should just pick one and stick to it.